packages/single-spa-welcome/package.json

```diff
@@ -30,6 +30,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.15.0",
+    "@babel/eslint-parser": "^7.15.0",
```

Comment:
This is required here: https://github.com/robmosca/create-single-spa/blob/9e99f853ae9ab6772450d5c14de7d027174daf26/packages/single-spa-welcome/.eslintrc#L3
I guess this was working before due to the different way the old version of pnpm was hoisting packages.